### PR TITLE
EP-66 Log of Pending UI

### PR DESCRIPTION
**Pull request by Scieon**
- This is only outgoing pending transfers not incoming

### CODE REVIEW

File: src/pages/transfers/transfers.html [hunk 1]
```diff
@@ -10,24 +10,23 @@
...
     <ion-segment [(ngModel)]="action">
-      <ion-segment-button value="pending" >
-        Pending
-      </ion-segment-button>
       <ion-segment-button value="history">
         History
       </ion-segment-button>
+      <ion-segment-button value="pending">
+        Pending
+      </ion-segment-button>
```
Comment (JTalarico) [inline src/pages/transfers/transfers.html]: Change this to Outgoing. Pending is more ambiguous .

File: src/pages/transfers/transfers.html [hunk 2]
```diff
@@ -10,24 +10,23 @@
...
     </ion-segment>
   </div>
 
   <div [ngSwitch]="action">
     <ion-list *ngSwitchCase="'pending'">
-      <ion-item>
```
Comment (JTalarico) [inline src/pages/transfers/transfers.html]: Same. Change to outgoing.

File: src/providers/transfer.service.ts [hunk 3]
```diff
@@ -54,4 +54,19 @@ export class TransferService {
...
+
+  /**
+   * Sends a get request to request pending transactions from a user
+   * @returns {Observable|"../../../Observable".Observable|"../../Observable".Observable}
+   */
+  public getPendingTransactions(){
+    return new Observable((observer) => {
+      this.http.get(environment.server_url + "/api/transfer/user/out")
+        .map((res) => res.json())
+        .subscribe((res) => {
```
Comment (JTalarico) [inline src/providers/transfer.service.ts:109]: Good use of observable. This makes unit testing much easier.

File: src/pages/transfers/transfers.ts [hunk 4]
```diff
@@ -27,13 +28,20 @@ export class TransfersPage {
...
+
   /**
    * Expands transaction to show more details
    */
-  public showTransferModal() {
-    let modal = this.modalCtrl.create(TransfersModalPage);
+  public showTransferModal(transfer) {
+    let modal = this.modalCtrl.create(TransfersModalPage, {transfer});
     modal.present().then();
   }
```
Comment (JTalarico) [inline src/pages/transfers/transfers.ts]: Why is there a .then() here? It serves no purpose.

**Review by JTalarico — DISMISSED** [review]

File: src/pages/transfers/transfers.ts (hunk 4 above)
Comment (ghiobi) [reply src/pages/transfers/transfers.ts]: Yes there's a purpose.
> Efforts and courage are not enough without purpose and direction. John F. Kennedy

Comment (ghiobi) [reply src/pages/transfers/transfers.ts]: Jk

**Review by ghiobi — APPROVED** [review]
Approved. :+1:

**Comment by Scieon** [comment]
Please don't merge this yet. I need to check if merge conflict was fixed properly.

**Pull request merged**